README.md

```diff
@@ -118,7 +95,7 @@ If your class extends an `abstract class` or `interface`, you can specify to bin
 ```kotlin
 interface MyInterface
 
-@SoloModule(MyInterface::class)
+@SoloModule(MyInterface::class, SingletonComponent::class)
```

Comment:
Having these named would make it more clear what the parameters are for in the documentation.